azure-pipelines.vrt-baseline.yml

```diff
@@ -15,6 +15,32 @@ variables:
 pool: '1ES-Host-Ubuntu'
 
 jobs:
+  # TODO: When web-components-v3 branch is merged with master, this file can be deleted and everything below this comment
```

Comment:
Time to delete this or do we need to wait until after this PR?

Reply:
I think we still need this, at least for now?